conf/modules.config

```diff
@@ -351,6 +351,10 @@ process {
         ]
     }
 
+    withName: KAIJU_KAIJU2KRONA {
+        ext.args = '-v -u'
```

Comment:
I wonder if `-u` shoudl be optional, but can leave for now

Reply:
It probably _should_, but under the test database we need `-u` to get output from the process.